ignite/templates/app/files/app/app.go.plush

```diff
@@ -839,6 +839,11 @@ func (app *App) RegisterTendermintService(clientCtx client.Context) {
 	)
 }
 
+// RegisterNodeService implements the Application.RegisterNodeService method.
+func (app *App) RegisterNodeService(clientCtx client.Context) {
```

Comment:
What the goal of implementing this interface ?

Reply:
After having a look at the feature in cosmos-sdk, it seems like the need to implement this method has been removed afterward in `release/0.46.x`, to prevent breaks with existing `Application`. 

That said, since the need to implement this method is required for next release, it's good to have it!